Makefile

```diff
@@ -384,8 +384,9 @@ endif
 
 	# Copy artifacts for upload to GitHub.
 	mkdir -p bin/github
-	$(foreach var,$(VALIDARCHES), cp bin/$(var)/calico bin/github/calico-$(var);)
-	$(foreach var,$(VALIDARCHES), cp bin/$(var)/calico-ipam bin/github/calico-ipam-$(var);)
+	$(foreach var,$(VALIDARCHES), cp bin/$(var)/install bin/github/install-$(var);)
```

Comment:
I think this only creates a binary called `install` (as opposed to `install-amd64`)

Reply:
Oh oops, misread the command, the destination should be `install-amd64`